app/models/concerns/hostext/search.rb

```diff
@@ -72,7 +72,7 @@ module Search
 
       scoped_search :relation => :reported_data, :on => :boot_time, :rename => 'boot_time', :only_explicit => true
       scoped_search :relation => :reported_data, :on => :boot_time, :rename => 'reported.boot_time', :only_explicit => true
-      scoped_search :relation => :reported_data, :on => :virtual, :rename => 'reported.virtual', :only_explicit => true
+      scoped_search :relation => :reported_data, :on => :virtual, :rename => 'reported.virtual', :complete_value => {:true => true, :false => false}
```

Comment:
This should have stayed `only_explicit` as well, to avoid inclusion in free-text searches

Reply:
Ah true, @domitea could you please open a PR that adds `:only_explicit => true` back to the definition? Feel free to use `Refs #32833` in the commit message.